src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -26,6 +26,7 @@ export default () => `
 						<li><a href="command:notebook.command.new">${escape(localize('welcomePage.newNotebook', "New notebook"))}</a></li>
 						<li class="mac-only"><a href="command:workbench.action.files.openLocalFileFolder">${escape(localize('welcomePage.openFileMac', "Open file"))}</a></li>
 						<li class="windows-only linux-only"><a href="command:workbench.action.files.openFile">${escape(localize('welcomePage.openFileLinuxPC', "Open file"))}</a></li>
+						<li><a href="command:azdata.resource.deploy">${escape(localize('welcomePage.deploySQL', "Deploy SQL Server…"))}</a></li>
```

Comment:
should the command include the string "SQL" in it?

Reply:
you mean "command:azdata.resource.deploy" should include SQL in it?

Reply:
@chlafreniere does my reply to Karl's comment make sense? if not, please let us know which SQL are you talking about :)